people/alexiamcdonald.html

```diff
@@ -0,0 +1,9 @@
+<<!DOCTYPE html>
+<html>
+<head>
+    <title>Alexia's Page</title>
+</head>
+<body>
+<p> yo I'm awesome! </p>
+</body>
+</html>
```

Comment:
This icon means that you are missing a newline at the end of your file. Git likes you to have new lines at the end! You can set up sublime to automatically add it by adding:

```json
"ensure_newline_at_eof_on_save": true
```

to your user preferences file.